datasets/meetbouten_azure/meetbouten/v2.0.0.json

```diff
@@ -29,13 +29,18 @@
         "type": "string",
         "description": "Beschrijving van de locatie van de meetbout, bijvoorbeeld 'in gemeenschappelijke muur'"
       },
-      "statusCode": {
-        "type": "integer",
+      "status": {
```

Comment:
Hier loopt onze spec. van het amsterdam schema voor op de werkelijkheid. Dit type nested velden wordt door de DSO API nog niet ondersteund. Zolang de DSO API niet gebruikt wordt, is er dus geen probleem. Maar, als dat op korte termijn de bedoeling is zullen wij die feature moeten gaan inbouwen. Staat op de backlog, maar nog niet erg vooraan.

Reply:
Dit is wat ons betreft geen probleem; we gebruiken de Amsterdam Schema bestanden zelf.

Reply:
Mooi, dat vermoedde ik al, maar ik wilde het even zeker weten :-)